home/ivp-planning/user-interface/the-3d-panel.md

```diff
@@ -1,28 +1,31 @@
 ---
 description: >-
-  The 3D-Panel acts primarily as visualization area. Like in the 2D-panel it
-  where you can place, move, scale and rotate machines and objects as if you
-  would work with a traditional floor plan on paper.
+  The 3D panel primarily serves as a visualization area. Similar to the 2D panel, it allows you to position, adjust size, and rotate machinery and objects as if you were working with a traditional paper floor plan.
```

Comment:
The 3D panel is not a visualization area. It allows planning just like the 2D panel.